### PR TITLE
Webseer tab list new schedule fix

### PR DESCRIPTION
**Pull request by unka65**
Fix schedule delete invalid function.
Do not list items on the Webseer tab before the schedule is created.  I added some text to explain why no items are listed.  Is it clear, or what would help?
Let me know if there is a better way for a new schedule and I will look into it.  Some thoughts are:
1. Change to not show the Devices and Webseer tabs until after the schedule is created (easy).
2. If possible to do, show the rows on the tabs for a new schedule.  When the Associate action is taken, add the schedule before associating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ be logged on GitHub.
 
 ## Changelog
 
+* issue#:15 Fix webseer tab to not show items before schedule is created
+* issue#:15 Fix schedule delete invalid function
```
Comment (netniV) [inline README.md]: There should not be two issues for the same number.  If the second is introduced by the first fix, it is merely a correct to the fix as long it is before a release.  After a release, a separate issue should raised to document the problem again.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ be logged on GitHub.
...
+
 * feature: webseer tab functional (webseer plugin update required to use schedule)
```
Comment (netniV) [inline README.md]: Feature should have an issue/pull request number assigned to them for clarity.

File: maint.php [hunk 3]
```diff
@@ -1054,7 +1056,15 @@ function clearFilter() {
 			form_end_row();
 		}
 	} else {
-		print "<tr><td colspan='8'><em>" . __('No Associated Devices Found', 'maint') . "</em></td></tr>";
+		if ($schedule_created) {
+			if (get_request_var('associated') == 'false') {
+				print "<tr><td colspan='8'><em>" . __('No devices found for selection', 'maint') . "</em></td></tr>";
+			} else {
+				print "<tr><td colspan='8'><em>" . __('No associated devices found for selection.  Uncheck Associated to list unassociated devices.', 'maint') . "</em></td></tr>";
```
Comment (netniV) [inline maint.php]: There is a duplication of the wording with the title, that seems superfluous.

File: maint.php [hunk 4]
```diff
@@ -1294,7 +1309,15 @@ function clearFilter() {
 			form_end_row();
 		}
 	} else {
-		print "<tr><td><em>" . __('No Associated WebSeer URL\'s Found', 'maint') . "</em></td></tr>";
+		if ($schedule_created) {
+			if (get_request_var('associated') == 'false') {
+				print "<tr><td colspan='8'><em>" . __('No webseer items found for selection', 'maint') . "</em></td></tr>";
+			} else {
+				print "<tr><td colspan='8'><em>" . __('No associated webseer items found for selection.  Uncheck Associated to list unassociated items.', 'maint') . "</em></td></tr>";
```
Comment (netniV) [inline maint.php]: Again see above for duplicate of text.

**Review by netniV — CHANGES_REQUESTED** [review]
Some minor changes but the work looks good otherwise.

File: maint.php (hunk 3 above)
Comment (unka65) [reply maint.php]: This has been changed.  Let me know if I misunderstood.

Comment (netniV) [reply maint.php]: New wording is much clearer, thanks!

**Review by netniV — APPROVED** [review]

**Pull request merged**